main.go

```diff
@@ -54,6 +54,7 @@ var (
 	concurrency int
 	timeout     time.Duration
 	csv         bool
+	reportcsv   bool
```

Comment:
What about calling it `csv-cum` as in cumulative csv?

Reply:
Updated.